app/controllers/admin/participants/induction_records_controller.rb

```diff
@@ -13,10 +16,35 @@ def show
       )
     end
 
+    def edit_preferred_email
+      authorize @induction_record
+
+      @participant_identities = @participant_profile.user.participant_identities
+    end
+
+    def update_preferred_email
+      authorize @induction_record
+
+      if @induction_record.update(induction_params)
```

Comment:
could we instead use the `Induction::ChangeInductionRecord` service to ensure this change creates a new induction record and the previous one is marked as changed??

Reply:
@cwrw any particular reason we would need that?

Reply:
I think usually induction records are audits to show the timeline of a profile, and if a status/school/provider/mentor id/schedule changes we create a new record and mark the previous as "changed". This also ensures the latest induction record is amended and not previous ones which might be out of date. Let me know your thoughts as to why not to do this

Reply:
@cwrw oh right. I don't mind creating a new IR, but we already amend IRs directly (i.e. when we defer/withdraw a training) or when we fix IRs created with the wrong preferred email or when we assign mentors to participants. 

So, unless the IR's preferred email should be protected due to business rules I'm unaware, then I think it's ok to amend the IR.